api/app/controllers/spree/api/v2/storefront/stores_controller.rb

```diff
@@ -3,12 +3,26 @@ module Api
     module V2
       module Storefront
         class StoresController < ::Spree::Api::V2::ResourceController
+          before_action :update_options
+
           def current
             render_serialized_payload { serialize_resource(current_store) }
           end
 
+          def export_google_rss_feed
+            send_data export_google_rss, filename: 'products.rss', type: 'text/xml'
+          end
+
           private
 
+          def update_options
```

Comment:
```suggestion
          def google_feed_settings
```
`update_options` here sounds like a method that you would call to update the options of the store. I would call this method `google_feed_settings` and call the instance variable `@google_feed_settings` as well to be clear. Instead of calling the method in `before_action`, you can call the method from within `export_google_rss` like this:
```ruby
def export_google_rss
  Spree::Dependencies.export_google_rss_service.constantize.new.call(google_rss_feed)
end
```
And then you don't need the `before_action`.